src/pages/release-announce/1.5.0.md

```diff
@@ -0,0 +1,192 @@
+---
+id: rn-1.5.0
+title: Release Announcement v1.5.0
+---
+
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Release Announcement v1.5.0
+We are pleased to announce that the Apache YuniKorn community has voted to release 1.5.0. 
+Apache YuniKorn is a standalone resource scheduler, designed for managing and scheduling Batch and Data workloads on container
+orchestration frameworks like Kubernetes for on-prem and on-cloud use cases.
+
+## Overview
+The Apache YuniKorn community has resolved 215 [JIRAs](https://issues.apache.org/jira/issues/?filter=12353274) in this release.
+
+Release manager: TingYao Huang
+
+Release date: 2024-03-01 (TBD)
+
+## Highlights
```

Comment:
Event streaming is also an important enhancement: https://issues.apache.org/jira/browse/YUNIKORN-2115

Reply:
Added in latest commit.